packages/eden/tests/links/internal/transformer.test.ts

```diff
@@ -8,7 +8,7 @@ import {
 
 describe('getDataTransformer', () => {
   test('returns default transformer for nullish input', () => {
-    const dataTransformer = getDataTransformer(undefined)
+    const dataTransformer = getDataTransformer()
```

Comment:
I guess it's the same thing, but I wanted to be explicit. Like all calls are meant to have the exact same output, which is what I'm testing.
```ts
  const dataTransformer = getDataTransformer()
  const dataTransformer = getDataTransformer(undefined)
  const dataTransformer = getDataTransformer(null)
```